<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryInterface</class>
 <widget class="QWidget" name="InventoryInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>322</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="twInventory">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>1001</width>
      <height>192</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>入库时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>类别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>商品名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>商品条码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>入库数量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>售出数量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>库存数量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>生产日期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>失效日期</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>20</y>
      <width>579</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbTime">
       <property name="text">
        <string>按照时间筛选：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QDateTimeEdit" name="dteTimeStart"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lbConnection">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QDateTimeEdit" name="dteTimeEnd"/>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pbtnTimeSelect">
       <property name="text">
        <string>筛选</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbSort">
       <property name="text">
        <string>按照类别筛选：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbsortSelect"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pbtnSortSelect">
       <property name="text">
        <string>筛选</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QLineEdit" name="leFuzzySearch">
       <property name="text">
        <string>商品模糊查询...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回销售界面</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>商品盘点界面</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
